libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -293,6 +297,9 @@ public void setAccountSwitcherActivityClass(Class<? extends AccountSwitcherActiv
      * @param name The name associated with the key.
      * @return The key used for encrypting salts and keys.
      */
+    /*
+     * TODO: Mark this deprecated and remove it in Mobile SDK 7.0.
+     */
     public String getKey(String name) {
```

Comment:
We need `getKey...` for now to perform a clean upgrade to `6.0`. After that, we can get rid of it. I don't want to mark it `deprecated` because app developers might accidentally remove it in `6.0` leaving no upgrade path.